packages/nimble/R/RCfunction_compile.R

```diff
@@ -269,6 +269,11 @@ RCfunProcessing <- setRefClass('RCfunProcessing',
                                        if(inherits(tryResult, 'try-error')) {
                                            stop(paste('There is some problem at the insertAdditions processing step for this code:\n', paste(deparse(compileInfo$origRcode), collapse = '\n'), collapse = '\n'), call. = FALSE)
                                        }
+
+                                       if(nimbleOptions('useRefactoredSizeProcessing')) {
+                                           exprClasses_setToEigenize(compileInfo$nimExpr, compileInfo$newLocalSymTab, compileInfo$typeEnv)
```

Comment:
At the moment, yes.  I think it will get slightly more complicated: If lifting temporaries out of expressions happens at later steps, we'll need to manage which pathway of catch-up processing needs to happen on those temporaries.